corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -341,6 +343,42 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             };
         },
     });
+    var paginateOptions = function (currentPage, totalPages) {
```

Comment:
Can you write unit tests for this function? You'll need to make this function public (that is, add it to the [returned functions](https://github.com/dimagi/commcare-hq/blob/148d87d8576e267c60c779ad0aeb9fe10ab73389/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L522)) so tests can access it and then make a new spec file with tests. [utils_spec.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/utils_spec.js) is an example of a small test file that just has a couple of unit tests for an independent function.